packages/web/components/place-limit-tool/index.tsx

```diff
@@ -562,6 +562,7 @@ export const PlaceLimitTool: FunctionComponent<PlaceLimitToolProps> = observer(
       swapState.isMarket,
       swapState.marketState.networkFeeError,
       swapState.marketState.isSlippageOverBalance,
+      account?.isWalletConnected,
```

Comment:
Why do we also remove this here? Is it related to something?

Reply:
It fixed a lint error

Reply:
Oh got it, thank you!